src/engine-actions/get-max-channel.js

```diff
@@ -8,16 +8,42 @@ const { listChannels } = require('../lnd-actions')
  * @return {String} res.maxBalance - the max balance in all open channels.
  */
 async function getMaxChannel ({ outbound = true } = {}) {
-  const { channels = [] } = await listChannels({ client: this.client })
+  const [
+    { channels = [] } = {},
+    { pendingOpenChannels = [] } = {}
+  ] = await Promise.all([
+    // The response from listChannels consists of channels that may be active or inactive
+    listChannels({ client: this.client }),
+    listPendingChannels({ client: this.client })
+  ])
   const balance = outbound ? 'localBalance' : 'remoteBalance'
 
-  if (channels.length === 0) {
-    this.logger.debug('getMaxChannel: No channels exist')
+  if (channels.length === 0 && pendingOpenChannels.length === 0) {
```

Comment:
[nit] A recent trick that I learned was `if (!channel.length && !pendingOpenChannels.length)`, since having a length of `0` will evaluate to false for JS